server/src/test/scala/com/cloudera/livy/server/recovery/ZooKeeperStateStoreSpec.scala

```diff
@@ -96,6 +96,21 @@ class ZooKeeperStateStoreSpec extends FunSpec with LivyBaseUnitTestSuite {
       }
     }
 
+    it("get should retrieve retry policy configs") {
+      withMock { f =>
+        mockExistsBuilder(f.curatorClient, true)
+
+        val getDataBuilder = mock[GetDataBuilder]
+
+        assert(f.stateStore.retryPolicy != null)
```

Comment:
Instead of assert, mind if you use `should`? When the check fails, `should` provides clear error messages.

`f.stateStore.retryPolicy should not be null`